### ISSUE TITLE
Submit your bulma PRs

### ISSUE THREAD

**Issue by Tofandel** [bulbacss/bulba#8]
You are welcome to resubmit your bulma PRs to Bulba

Since this project is not a direct github fork of bulma you will need to submit your pull request to https://github.com/bulbacss/bulma which is a direct fork and mirrors this repository

To do so just go on your PR branch and create a new PR and change the base repository
![image](https://user-images.githubusercontent.com/6115458/91047563-90bc0200-e61a-11ea-913b-d5bf0a26698c.png)

We will evaluate the quality of the PR and request changes accordingly

We will likely request some changes because of atemizer or you may have merge conflicts if you were modifying previous functionalities but you'll have a way higher chance of seeing your PR merged here than on Bulma

Once your PR is merged on the bulbacss/bulma repository, they will be synced to this repository

**Comment by Tofandel** [comment]
https://github.com/jgthms/bulma/issues/2966 - https://github.com/bulbacss/bulma/pull/1
